patches/minecraft/net/minecraft/world/item/ItemStack.java.patch

```diff
@@ -144,6 +144,10 @@
     }
  
     public boolean m_41629_(int p_41630_, Random p_41631_, @Nullable ServerPlayer p_41632_) {
++      if (net.minecraftforge.event.ForgeEventFactory.onPlayerDamageItem(p_41632_, this, this.m_41776_() - this.m_41773_(), this.m_41776_() - this.m_41773_() - p_41630_)) return false;
```

Comment:
Why not pass in the damage value and the new damage value (which is what everything in Minecraft uses) instead of the inverted one ("durability")?